pkg/catalog/catalog.go

```diff
@@ -1548,6 +1655,7 @@ func (c *Catalog) Close() error {
 			_ = multierror.Append(errs, err)
 		}
 	}
+	c.workpool.StopAndWaitFor(1 * time.Second)
```

Comment:
Seems like 1 second might be a bit strict

Reply:
Ping

Reply:
Changed to 5 seconds.